test/node/connection-inspect.js

```diff
@@ -38,9 +41,9 @@ test.test('must process inspect messages', (test) => {
     expectedInterfaces.forEach((iface) => {
       connection.inspectInterface(iface, (error, methods) => {
         test.assertNot(error, `must inspect ${iface}`);
-        Object.keys(app.interfaces[iface]).forEach(method =>
+        Object.keys(app.interfaces[iface]).forEach(method => (
```

Comment:
Ditto, I agree with the linter on this one. Either this, or the function itself should be moved to a new line if we allow line-breaking arrow functions without parens. I find the left-hand side of the diff a bit weird to read.